pandas/tests/series/test_operators.py

```diff
@@ -741,6 +741,21 @@ def test_op_duplicate_index(self):
         expected = pd.Series([11, 12, np.nan], index=[1, 1, 2])
         assert_series_equal(result, expected)
 
+    def test_divmod(self):
+        # GH25557
+        a = Series([1, 1, 1, np.nan], index=['a', 'b', 'c', 'd'])
+        b = Series([2, np.nan, 1, np.nan], index=['a', 'b', 'd', 'e'])
+
+        r1 = divmod(a, b)
```

Comment:
use the format
result=
expected=
assert_series_equal

Reply:
Fixed.